run_tests.py

```diff
@@ -155,30 +156,36 @@ def prompt_user_to_run_tests(test_names):
         elif answer.upper()[0] == "Y":
             return
 
+def runTestProto(self):
+    """
+    Function to be used as basis for runTest function of ReportFailLoadTestsuiteTestCase.
+    We create a specific named test function to improve the test failure summary details.
+    """
+    self.fail(self.msg)
 
 class ReportFailLoadTestsuiteTestCase(unittest.TestCase):
     """
     Class to allow reporting of an error to run any tests.
 
     Args:
-        msg: Error message.
+        func_name:   Name of function to create for the test.
+        msg:         Error message explaining test failure.
 
     Returns:
         None
     """
-    def __init__(self, msg):
-        super(ReportFailLoadTestsuiteTestCase,self).__init__()
+    def __init__(self, func_name, msg):
+        setattr(self, func_name, types.MethodType(runTestProto, self))
```

Comment:
change func_name to failing_module_name then
func_name = "test_start_{}_test_environment_and_ioc"
setattr(self, func_name, self.runTestProto)

Reply:
You might be able to get away with not using init(funcname

Reply:
> You might be able to get away with not using init(funcname
Sorry, I don't follow - or is the suggestion to create a new class dynamically instead so the  function nane doesn't need to change?